webrtc/simplecall-no-ssrcs.https.html

```diff
@@ -9,7 +9,7 @@
   <div id="log"></div>
   <div>
     <video id="local-view" muted autoplay="autoplay"></video>
-    <video id="remote-view" muted autoplay="autoplay"/>
+    <video id="remote-view" muted autoplay="autoplay"></video>
     </video>
```

Comment:
False alarm here and in the other video files

Reply:
Good catch, fixed